package.json

```diff
@@ -3,7 +3,8 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "start": "react-native start"
+    "start": "react-native start",
+    "x": "open ios/NeverMissA.xcodeproj"
```

Comment:
`npm` can run commands for you, which is a good place to store project-specific commands. With this change running `npm run x` from the command line will open the xcode project, so you don't have to hunt for it.